src/test/java/core/basesyntax/strategy/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static final FruitTransaction.Operation BALANCE = FruitTransaction.Operation.BALANCE;
```

Comment:
```suggestion
    private static final FruitTransaction.Operation BALANCE_OPERATION = FruitTransaction.Operation.BALANCE;
```
the same for other similar constants